src/models/FruitStorage.ts

```
@@ -0,0 +1,26 @@
+  id: { type: Schema.Types.ObjectId },
+  fruitId: { type: Schema.Types.ObjectId },
+  limit: { type: Number },
+  count: { type: Number }
+});
+
+FruitStorageSchema.set('toJSON', {
+  virtuals: true,
+  versionKey: false,
+  transform(_: any, ret: any) {
+    ret.id = ret._id; // eslint-disable-line no-param-reassign
+    delete ret._id; // eslint-disable-line no-param-reassign
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)